### PR TITLE
Adds new image for Mandrel's native-image executable

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

**Comment by zakkak** [comment]
Waiting for approval before editting `.github/workflows/push-images.yml`.

\cc @galderz

**Comment by cescoffier** [comment]
Before merging, can you update the `.github/workflows/push-images.yml` file so we get the deployment setup too.

File: .github/build-mandrel-images.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+for version in "${VERSIONS[@]}"
+do
+    echo "Generating ${PREFIX_NAME}:${version}"
+    cekit --descriptor ${IMAGE} build \
+        --overrides "{'version': '${version}', 'modules': {'install': [{'name':'mandrel', 'version': '${version}'}]}}" \
+        ${BUILD_ENGINE} --tag="${PREFIX_NAME}:${version}"
+done
+
+docker image prune -f
```
Comment (jerboaa) [inline .github/build-mandrel-images.sh:20]: Shouldn't this be `${BUILD_ENGINE}` instead of `docker`?

Comment (zakkak) [reply .github/build-mandrel-images.sh:20]: Good catch @jerboaa the same "issue" appears in https://github.com/quarkusio/quarkus-images/blob/master/.github/build-native-images.sh (where I copied this from).

File: .github/build-mandrel-images.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+PREFIX_NAME=quay.io/quarkus/ubi-quarkus-mandrel
+IMAGE=quarkus-mandrel.yaml
+VERSIONS=('20.1.0.0.Alpha1-java11')
+BUILD_ENGINE=podman
```
Comment (zakkak) [inline .github/build-mandrel-images.sh:7]: Changing this to docker

File: .github/build-mandrel-images.sh (hunk 1 above)
Comment (zakkak) [reply .github/build-mandrel-images.sh]: This is now resolved (for mandrel images only though)

**Comment by zakkak** [comment]
@cescoffier thanks for the review. I don't have write access to the repository though.
Could you please merge this for me or give me write access?

**Comment by zakkak** [comment]
> Before merging, can you update the `.github/workflows/push-images.yml` file so we get the deployment setup too.

This is now updated as well

**Comment by cescoffier** [comment]
Waiting for CI before merging. Should be merged this afternoon.

File: .github/build-mandrel-images.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+PREFIX_NAME=quay.io/quarkus/ubi-quarkus-mandrel
```
Comment (maxandersen) [inline .github/build-mandrel-images.sh:4]: shuoldn't we refrain from using ubi-* prefix as this isn't a real ubi image ? 

p.s. I would check our other names but for the moment I can't access quay.io for some unknown reason ;/

Comment (maxandersen) [reply .github/build-mandrel-images.sh:4]: just checked the sources and we already use ubi-* thus never mind ;)

**Comment by zakkak** [comment]
@cescoffier thanks for merging this. Could you also please trigger a push to quay.io so that we don't have to wait for the scheduled (in 4 days) push?

**Comment by cescoffier** [comment]
Sure. 

In general, we push new versions every Sunday (with updated parent images). We only push in between when we do changes. 

So, I will push the images in a few minutes and then it would be done again on Sunday (and next Sunday...)

**Comment by cescoffier** [comment]
Push in progress: https://github.com/quarkusio/quarkus-images/actions/runs/144936237

**Comment by zakkak** [comment]
Thanks :)

**Pull request merged**